modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -68,7 +69,9 @@ func (k *Keeper) sendPacket(
 		return 0, "", err
 	}
 	// check if packet is timed out on the receiving chain
-	timeout := channeltypes.NewTimeoutWithTimestamp(timeoutTimestamp)
+	// convert packet timeout to nanoseconds for now to use existing helper function
+	// TODO: Remove this workaround with Issue #7414: https://github.com/cosmos/ibc-go/issues/7414
```

Comment:
cc: @damiannolan

Reply:
For a second I thought this was converting to seconds instead of nano seconds, but I see how the construction is here. Just had to check out locally to make sure. 

Thanks! I'll pull this into the timeout refactor branch I had open in draft!